### DISCUSSION TITLE
Does cockpit support yaml or helm installation for k8s?

### DISCUSSION THREAD

**Discussion by GeekDot** [cockpit-project/cockpit#18692] [Q&A] [answered]
Does cockpit support yaml or helm installation for k8s? Is there a Docker image?

**Comment by martinpitt** [comment, marked as answer]
Yes, you can run cockpit's web server as a container. See https://quay.io/repository/cockpit/ws . With that, you can log into any SSH target that is reachable from that container and has cockpit-system instaled to get a cockpit UI.

**Reply by GeekDot** [reply]
Thank you so much.
